src/CertifiedData.mo

```diff
@@ -32,7 +32,9 @@ module {
   ///
   /// See a full example on how to use certified variables here: https://github.com/dfinity/examples/tree/master/motoko/cert-var
   ///
-  public let set : (data : Blob) -> () = Prim.setCertifiedData;
+  public func set<system>(data : Blob) : () {
+    Prim.setCertifiedData(data)
```

Comment:
I guess we may need to make the Prim.setCertifiedData take a <system> argument too to make this really safe (otherwise people can directly call Prim.setCertifiedData).

Reply:
> I guess we may need to make the Prim.setCertifiedData take a argument too to make this really safe

This is probably worth doing, since otherwise there isn't much of a benefit to changing the base library if people can go around it by importing `Prim` directly. 

I'll roll this back to `public let set : (data : Blob) -> () = Prim.setCertifiedData` so that it passes on whether the prim function has the `<system>` requirement.